examples/reliable_conversation/src/utils/test_runner.py

```
@@ -0,0 +1,289 @@
+            if passed:
+                self.console.print(f"  [green]✓ {name}[/green]")
+            else:
+                self.console.print(f"  [red]✗ {name}[/red]")
+                if details:
+                    self.console.print(f"    [dim]{details}[/dim]")
+    
+    def display_summary(self):
+        """Display final test summary"""
+        total_time = time.time() - self.start_time
+        
+        self.console.print(f"\n[bold blue]━━━ Test Summary ━━━[/bold blue]\n")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)